packages/cozy-procedures/src/components/Duration.jsx

```diff
@@ -1,22 +1,102 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import get from 'lodash/get'
 import Topbar from './Topbar'
-import { translate } from 'cozy-ui/transpiled/react/I18n'
+import {
+  translate,
+  Title,
+  SubTitle,
+  Label,
+  Button,
+  Card
+} from 'cozy-ui/transpiled/react'
+import Slider from '@material-ui/lab/Slider'
+import { withRouter } from 'react-router'
+import creditApplicationTemplate from '../templates/creditApplicationTemplate'
+
+class Duration extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.confirm = this.confirm.bind(this)
+  }
 
-class Duration extends React.Component {
   render() {
-    const { t } = this.props
+    const { t, duration, amount, updateDuration } = this.props
+
+    const min = get(creditApplicationTemplate, 'procedureData.duration.min')
+    const max = get(creditApplicationTemplate, 'procedureData.duration.max')
+    const defaultDuration = get(
+      creditApplicationTemplate,
+      'procedureData.duration.default'
+    )
+    const defaultAmount = get(
+      creditApplicationTemplate,
+      'procedureData.amount.default'
+    )
+
+    const simulationDuration = duration || defaultDuration
```

Comment:
why can't we set the default duration in the initial state?

Reply:
Because then the Overview screen would never show the "ghost" duration button.